net/ampr-ripd/files/ampr-ripd-init

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh /etc/rc.common
+
+START=95
+STOP=10
+
+EXTRA_COMMANDS="configure"
+EXTRA_HELP="        configure       Configure service parameters"
+
+start() {
+	default_addr="44.127.254.254"
```

Comment:
I wonder. This IP can be configured, right? Why it is not in config file and you do it in init script?
Also, this IP address is created by uci-defaults.

I find this confusing, though.

Reply:
Kind reminder here as well. :)